src/components/CActorArgument.vue

```diff
@@ -0,0 +1,296 @@
+<template>
+  <div class="w-full md:w-full py-2 mb-6 md:mb-0">
+    <label class="block uppercase tracking-wide text-gray-700 text-xs font-bold mb-2">
```

Comment:
this style is the same as `actor` or `options`, we should just create a `parameters` section, and then apply the same style between parameters and options. In my opinion, it would be better if the options would be modifiable, but I like the style on only one line.